castai/data_source_castai_eks_cluster_id.go

```diff
@@ -0,0 +1,68 @@
+package castai
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/castai/terraform-provider-castai/castai/sdk"
+)
+
+const (
+	EKSClusterIDFieldAccountId = "account_id"
```

Comment:
I wonder if we can have generic data source for getting cluster id. It can be useful for other cloud providers as well.

Reply:
It might be useful, but now we don't have any need for that so can be skipped I think